jeoparpy/ui/maingame/podiapanel.py

```diff
@@ -85,15 +92,15 @@ def _init_podia(self, gameData, scalar):
         fonts = (('team1', 42), ('team2', 33), ('team3', 40), ('team1', 42), ('team2', 33))
         fonts = tuple((FONTS[n], s) for n,s in fonts)
 
-        for i in range(5):
+        for i in range(PLAYER_NUM):
             p = Podium(i, img, scalar, gameData.players[i].name,
                        fonts[i], nameBounds, podia)
 
         return self._position_podia(podia)
 
     def _position_podia(self, podia):
-        ph = podia.sprites()[0].rect.h - 10
```

Comment:
It looks like this was where the issue with the timer being off screen was (#11)